projects/kit/utils/mask/create-auto-corrected-time-pipe.ts

```diff
@@ -1,23 +1,24 @@
 import {TuiTimeMode} from '@taiga-ui/cdk';
 import {TuiTextMaskPipeHandler} from '@taiga-ui/core';
+import {TuiTimeFormatParts} from '@taiga-ui/kit/types';
 
 /**
  * Adjusts the entered time by omitting only suitable values for hours and minutes
  * @returns time as a string
  */
 export function tuiCreateAutoCorrectedTimePipe(
     timeMode: TuiTimeMode = 'HH:MM',
+    maxValues: Record<TuiTimeFormatParts, number> = {HH: 23, MM: 59, SS: 59, MS: 999},
```

Comment:
`{HH: 23, MM: 59, SS: 59, MS: 999}` maybe move to special constant for reuse?

Reply:
Also thought about it. Thanks for supporting this idea